pss/trojan/message.go

```diff
@@ -129,6 +124,25 @@ func Unwrap(c chunk.Chunk) (*Message, error) {
 	return m, err
 }
 
+// IsPotential returns true if the given chunk is a potential trojan
+func IsPotential(c chunk.Chunk) bool {
```

Comment:
The name `IsPotential` is a bit misleading in my humble opinion.
Maybe `ProbableChunk` could be a better name to describe what the function is doing.
Just a suggestion for the naming, the function itself is OK.

Good work on the comments.

Reply:
i think it makes a bit more sense when you include the package in the name, i.e. `trojan.IsPotential(c Chunk)`, but point taken :+1: